src/operator/contrib/adamw-inl.h

```diff
@@ -151,7 +135,7 @@ struct MPAdamWUpdate {
                const std::vector<TBlob> &outputs,
                const float rescale_grad) {
     using namespace mxnet_op;
-    AdamWParam param = nnvm::get<AdamWParam>(attrs.parsed);
+    const auto &param = nnvm::get<AdamWParam>(attrs.parsed);
```

Comment:
Put the `&` next to `auto` (so `const auto& param` instead of `const auto &param`) to be consistent with the rest of the code.

Reply:
I could do that, but it does not look like a standard for the whole MxNet, perhaps, just for `adamw-inl.h`. I searched for `auto &` and `auto&` in all `*h, *.cc, *.cu` files. As I could see, the probability of this or that option is approximately 50%.

Reply:
I understand, but inside a single file we should try to make it consistent :-).

Reply:
OK, I will change that.